From e47385ab0fc6ff5d22721677b97c39dc745fb4b9 Mon Sep 17 00:00:00 2001
From: Joshua Mack <jmack2545@arizona.edu>
Date: Fri, 28 Apr 2023 21:13:33 +0000
Subject: [PATCH] Actually disable ldconfig

---
 src/libpfm4/config.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/libpfm4/config.mk b/src/libpfm4/config.mk
index 41fa091b3..7f0b8980e 100644
--- a/src/libpfm4/config.mk
+++ b/src/libpfm4/config.mk
@@ -163,7 +163,6 @@ endif
 
 ifeq ($(ARCH),aarch64)
 CONFIG_PFMLIB_ARCH_ARM64=y
-LDCONFIG=true
 endif
 
 ifeq ($(ARCH),arm64)
@@ -209,6 +208,10 @@ DBG?=-g -Wall -Werror
 LDCONFIG=true
 endif
 
+ifeq ($(ARCH),aarch64)
+LDCONFIG=true
+endif
+
 ifeq ($(SYS),FreeBSD)
 # gcc-4.2 does not like -Wextra and some of our initialization code
 DBG=-g -Wall -Werror
